topics/safe-data-transfer.qmd

```diff
@@ -0,0 +1,40 @@
+* [SURFfilesender](https://www.surf.nl/en/surffilesender-send-large-files-securely-and-encrypted): as a SURFfilesender user, you can send a voucher to someone who doesn’t have access to this tool. This person can use this voucher to send documents to you. These files can be encrypted.
+* 🔒 [Zivver i](https://services.vu.nl/esc?id=emp_taxonomy_topic&topic_id=01d4a69797cf09d0e553359fe153afdd)s an email plugin with which you can encrypt emails and attachments.
+
+#### Offline data outside campus
+
+If you are doing fieldwork in an area with limited internet access, you might use a portable device to initially store your data during the phase of data collection, such as a USB drive or an external hard drive. These data can be transferred to a storage location that is connected to the internet (e.g. G-drive, SURFdrive) later. Please make sure that the data on such portable devices are secured, by using encryption (and by transporting them safely by using a lockable briefcase or backpack).
+
+### Transporting physical data
+
+If physical objects need to be transported, you should check with the data manager at your department (if available) what options are available. Special briefcases that can be locked or secure backpacks may need to be used to keep informed consent forms or other sensitive data objects (USB drives etc.) secure during transport.
+
+### Data transportation and transfer across borders
+
+Some countries have rules to control the movement of encryption technology that enter or exit their borders. If you need to travel with an encrypted laptop to secure your data, for example during fieldwork abroad, please keep this in mind. If you need to transfer data in and out of such countries, please get advice on encryption and secure transportation at the [IT Service Desk](mailto:servicedesk.it@vu.nl).
+
+If you have general questions about how to protect your data when transporting or transferring them, you can contact the [IT Service Desk](mailto:servicedesk.it@vu.nl). In case of complex situations for which you need tailored support, you can consult the IT Relationship Manager representing the research domain, who can request capacity at IT for setting up an information security plan. Such a plan is usually based on documents which need to be completed beforehand, like a [Data Protection Impact Assessment](https://ubvu.github.io/open-handbook/guides/plan-and-design.html#what-is-a-dpia) and a [Data Classification](https://ubvu.github.io/open-handbook/guides/plan-and-design.html#data-classification). Please note that IT-capacity for tailored support is a paid service for which budget needs to be reserved.
```

Comment:
```suggestion
Some countries have rules to control the movement of encryption technology that enter or exit their borders. If you need to travel with an encrypted laptop to secure your data, for example during fieldwork abroad, please keep this in mind. If you need to transfer data in and out of such countries, please get advice on encryption and secure transportation at the [IT Service Desk](mailto:servicedesk.it@vu.nl).

### Support

If you have general questions about how to protect your data when transporting or transferring them, you can contact the [IT Service Desk](mailto:servicedesk.it@vu.nl). In case of complex situations for which you need tailored support, you can consult the IT Relationship Manager representing the research domain, who can request capacity at IT for setting up an information security plan. Such a plan is usually based on documents which need to be completed beforehand, like a [Data Protection Impact Assessment](https://ubvu.github.io/open-handbook/guides/plan-and-design.html#what-is-a-dpia) and a [Data Classification](https://ubvu.github.io/open-handbook/guides/plan-and-design.html#data-classification).
```